project/GameMaster.py

```diff
@@ -33,6 +32,21 @@ def test_login_bad_username(self):
         self.assertFalse(isadmin, "Admin flag incorrectly set")
         self.assertEqual("", username, "Username has value when it should be empty")
 
+class TestEndGame(unittest.TestCase):
```

Comment:
All these tests should go in game I think. Since game is now in CLI GameMaker doesn't really have direct access to it. The CLI will verify that the current user is an admin and then allow the Game.end_game() call.